core/domain/services/admin/events/editor/EventManagers.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace EventEspresso\core\domain\services\admin\events\editor;
+
+use EventEspresso\core\domain\services\graphql\Utilities;
+
+/**
+ * Class EventManagers
+ *
+ * @author  Brent Christensen
+ * @package EventEspresso\core\domain\services\admin\events\editor
+ * @since   $VID:$
+ */
+class EventManagers implements EventEditorDataInterface
+{
+
+    /**
+     * @var Utilities
+     */
+    private $utilities;
+
+
+    /**
+     * EventManagers constructor.
+     *
+     * @param Utilities $utilities
+     */
+    public function __construct(Utilities $utilities)
+    {
+        $this->utilities = $utilities;
+    }
+
+
+    /**
+     * @param int $eventId
+     * @return array
+     */
+    public function getData(int $eventId)
+    {
+        global $wpdb, $wp_roles;
+        $roles = $wp_roles->role_objects;
+        $capabilities = apply_filters(
+            'FHEE__EventEspresso_core_domain_services_admin_events_editor_EventManagers__getData__capability',
+            ['ee_edit_events', 'ee_edit_event'],
+            $roles
+        );
+        $SQL = "SELECT u1.ID, u1.display_name FROM {$wpdb->users} AS u1 "
+               . "INNER JOIN {$wpdb->usermeta} AS u2 ON u1.ID = u2.user_id "
+               . "AND u2.meta_key='{$wpdb->prefix}capabilities' "
+               . 'WHERE';
```

Comment:
This may not work because all the capabilities are not added to user meta

Reply:
ahhhh... but I'm not getting the actual caps from the user_meta table... I'm getting them from the WP_Role objects, then checking for the roles that have the desired caps

Reply:
I should add some comments to the code to make things clear